core/setup.py

```diff
@@ -75,7 +75,7 @@
         "minimal-snowplow-tracker>=0.0.2,<0.1",
         "dbt-semantic-interfaces>=0.5.1,<0.6",
         # Minor versions for these are expected to be backwards-compatible
-        "dbt-common>=1.0.1,<2.0",
+        "dbt-common>=1.0.2,<2.0",
```

Comment:
The addition of the attr `silence` to the class `WarnErrorOptions` is new, and got released in dbt-common `1.0.2`. If we don't bump it, the minimum then it would be possible for us to get dbt-common `1.0.1` which doesn't support silence, and we'd break at run time. [Here is the work in dbt-common](https://github.com/dbt-labs/dbt-common/pull/112) which added the functionality

Reply:
I started a thread in slack about this.  Unrelated to you actual PR - more around how this should have been a minor bump instead of a patch.